scripts/divi-icu-beds.py

```diff
@@ -50,6 +52,8 @@
 meta["unix_timestmap"] = unix_utc
 meta["iso_timestamp"] = iso_utc
 
+mongoDBUpload.insertData(states, meta)
```

Comment:
I would be afraid of this script to fail if the mongoDB is not accessible.
how about you move all mongo-related things to scripts/mongoDBUpload.py ?
there you could e.g. read in the (alphabetically) latest JSON in data and upload then.
Then we could schedule that mongo action after the autocommit.

How does that sound to you? We're open for discussion :-)

Reply:
i just thought we would reduce overhead of parsing hear

Reply:
would be try catch ok ?

Reply:
ok for me, thanks